internal/kgateway/reports/reporter_test.go

```diff
@@ -14,8 +14,13 @@ import (
 	gwv1a2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
 
 	"github.com/kgateway-dev/kgateway/v2/internal/kgateway/reports"
+	"github.com/kgateway-dev/kgateway/v2/internal/kgateway/wellknown"
 )
 
+const fake_condition = "kgateway.dev/SomeCondition"
```

Comment:
[nit] for consistency
```suggestion
const fakeCondition = "kgateway.dev/SomeCondition"
```